src/types/plugin-input.ts

```diff
@@ -23,12 +25,20 @@ export const startStopSchema = T.Object({
   ),
   miscellaneous: T.Object(
     {
-      maxConcurrentTasks: T.Number(),
-      startRequiresWallet: T.Boolean(),
+      startRequiresWallet: T.Boolean({ default: true }),
+      maxConcurrentTasks: T.Record(T.String(), T.Integer(), { default: { admin: 20, member: 10, contributor: 2 } }),
```

Comment:
Most likely we can make sure that the Integer value is greater than `0` because it wouldn't make sense.